test/server/utils/new-line-replacer.js

```diff
@@ -0,0 +1,3 @@
+exports.replace = function (content) {
```

Comment:
But the best thing will be to not extract utilities at all, just keep this short functions in both files.